docs/reference/commandline/dockerd.md

```diff
@@ -977,3 +977,39 @@ has been provided in flags and `cluster-advertise` not, `cluster-advertise`
 can be added in the configuration file without accompanied by `--cluster-store`
 Configuration reload will log a warning message if it detects a change in
 previously configured cluster configurations.
+
+
+## Running multiple daemons
+
+From docker 1.9.1 it is possible to run two Docker daemons on a single host.
+Setting the following `docker daemon`` flags resolves conflicts if two or more daemons are set to use the defaults:
+```
```

Comment:
Can you add a blank line before this, and add a "language" hint (it's not strictly "bash", but it prevents highlightjs from being smart and seeing it as a "comment" :smile:. Also, there's no need to indent the flags/options here, so can you remove the two-space indentation?

	```bash
	  -b, --bridge=                        Attach containers to a network bridge
	  --exec-root=/var/run/docker          Root of the Docker execdriver
	  -g, --graph=/var/lib/docker          Root of the Docker runtime
	  -p, --pidfile=/var/run/docker.pid    Path to use for daemon PID file
	```